docs/spec/data-plane.md

```diff
@@ -91,6 +91,14 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### Event reply header
+
+An event sender, including Source and Broker and Channel, SHOULD include
+the event reply header `K-Eventing-Event-Reply: response` in an event delivery
```

Comment:
`K-Eventing-Event-Reply` is a funny stutter. I would expect `K-Event-Consumer-Mode` or something like that.

Reply:
In the [feature track doc](https://docs.google.com/document/d/1ELQtKOMz6T5vTXbWp30E8r6oLYOXPjWacv4AVRi36-w/edit) this header is called `K-Eventing-Accept-Reply`. I think this might be a typo.

I like `Accept` in the name because it suggests the same semantics and format as the standard HTTP Accept-* headers.

Reply:
There was a typo, it was actually `K-Eventing-Accept-Reply` as in the doc, does this name make more sense?